### DISCUSSION TITLE
Two Strimzi MirrorMaker2 instances with the same target cluster

### DISCUSSION THREAD

**Discussion by EmiliaNowakowska** [strimzi/strimzi-kafka-operator#6630] [Q&A] [answered]
Hello,

We would like to replicate topics from 2 different source clusters into one target cluster. The topics are different on each of the source clusters.
Our idea was to have 2 Strimzi MirrorMaker2 instances (one for each source cluster).
We have separate namespace on our EKS cluster for each MirrorMaker2 instance. 
We are using replication.policy.class: "org.apache.kafka.connect.mirror.IdentityReplicationPolicy" to avoid aliases for topics.

One instance is running fine. When we apply the second MirrorMaker2 config, pods are up and running but replication is not working. 

Is it possible to run 2 MirrorMaker2 instances for the same target cluster? 
Is the problem that both MirrorMaker2 instances are trying to use the same internal topics on target cluster?:
```
mirrormaker2-cluster-offset-syncs
mirrormaker2-cluster-status
mirrormaker2-cluster-configs
mirrormaker2-cluster-offsets
heartbeats
```

Could you please suggest how to replicate topics from 2 different source clusters into the same target cluster with Strimzi MirrorMaker2 config?
Should we use one MirrorMaker2 instance?

**Comment by scholzj** [comment, marked as answer]
You should be able to have multiple source clusters with single target cluster even in one MM2 resource I think. If you for some reason want two of them, you would need to configure these topics so that each of the MM2 instances is using different ones. That should be possible for the Connect based topics. But not sure about the internal MM2 topics. I think the ability to configure their names were added only recently in Apache Kafka.

**Reply by EmiliaNowakowska** [reply]
Thank you Jakub for the quick answer. 
I will try to use one MM2 resource for both source clusters. Hope it will work.
